src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -404,30 +405,32 @@ private void updateSupplementaryEvidence(
             );
         }
 
-        List<Map<String, Object>> newCaseDocuments = attachExceptionRecordReference(
-            callBackEvent.exceptionRecordDocuments,
-            callBackEvent.exceptionRecordId
-        );
+        if (erDocumentConnectives.hasMissing()) {
+            List<Map<String, Object>> newCaseDocuments = attachExceptionRecordReference(
```

Comment:
In a future PR makes sense to wrap this block into a method call something like 'attachNewDocuments'

Reply:
lot's of refactoring is screaming around these services and whatnot. best to leave such activity as a separate action once changes in particular areas have cooled down and investigation + refactoring can be done